benchmark_attacks.py

```diff
@@ -31,23 +35,59 @@ def mel_noise_attack(epsilon, data, sr, n_fft, hop_length, n_mels):
     )
     return reconstructed_audio
 
-def band_pass_attack(data, sr, low, high):
```

Comment:
Why did we remove this function? Even if it isn't good, it could be still a comparison point.